packages/Checkbox/stories/Checkbox.stories.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withKnobs } from "@storybook/addon-knobs";
+import { CheckboxStory } from "./Checkbox.stories.styles";
+import CheckboxExample from "./examples/CheckboxExample";
+import ShowcaseStory from "./examples/Showcase";
+
+storiesOf("Checkbox", module)
+  .addDecorator(withKnobs)
+  .add("Showcase", ShowcaseStory);
+
+storiesOf("Checkbox/Automation Tests/Screener", module).add("CheckboxExample", () => (
+  <CheckboxStory>
+    <CheckboxExample size="large" />
```

Comment:
We can clean up the Storybook stories later, but why make the Screener story "large"?